internal/provisioner/objects/dataplane/dataplane.go

```diff
@@ -341,9 +341,10 @@ func DesiredDaemonSet(contour *model.Contour, contourImage, envoyImage string) *
 
 	ds := &appsv1.DaemonSet{
 		ObjectMeta: metav1.ObjectMeta{
-			Namespace: contour.Namespace,
-			Name:      contour.EnvoyDataPlaneName(),
-			Labels:    contour.AppLabels(),
+			Namespace:   contour.Namespace,
+			Name:        contour.EnvoyDataPlaneName(),
+			Labels:      contour.WorkloadLabels(),
+			Annotations: contour.CommonAnnotations(),
```

Comment:
It _probably_ makes sense to propagate these annotations to the pod template as well - thoughts?

Reply:
yeah seems right to me to propagate these down in addition to the other existing labels/annotations we place on the pod template(s)